src/vma/proto/dst_entry_tcp.cpp

```diff
@@ -107,13 +109,26 @@ ssize_t dst_entry_tcp::fast_send(const iovec* p_iov, const ssize_t sz_iov, bool
 		/* update L3(Total Length) with total size of L3 header, TCP header and data */
 		p_pkt->hdr.m_ip_hdr.tot_len = (htons)(p_tcp_iov[0].iovec.iov_len + m_header.m_ip_header_len);
 
-		m_p_send_wqe = (total_packet_len < m_max_inline ? &m_inline_send_wqe : &m_not_inline_send_wqe);
-
-		m_sge[0].addr = (uintptr_t)((uint8_t*)p_pkt + hdr_alignment_diff);
-		m_sge[0].length = total_packet_len;
-		m_p_send_wqe->wr_id = (uintptr_t)p_tcp_iov[0].p_desc;
-		p_tcp_iov[0].p_desc->tx.p_ip_h = &p_pkt->hdr.m_ip_hdr;
-		p_tcp_iov[0].p_desc->tx.p_tcp_h =(struct tcphdr*)((uint8_t*)(&(p_pkt->hdr.m_ip_hdr))+sizeof(p_pkt->hdr.m_ip_hdr));
```

Comment:
We will have issues when tx csum is not supported.

Reply:
can you explain.